parts/k8s/kubernetesagentresourcesvmas.t

```diff
@@ -25,6 +25,7 @@
       "location": "[variables('location')]",
       "name": "[concat(variables('{{.Name}}VMNamePrefix'), 'nic-', copyIndex(variables('{{.Name}}Offset')))]",
       "properties": {
+        "enableAcceleratedNetworking" : true,
```

Comment:
Is this something we might want to consider putting behind a feature flag for now? Ie. have an optional `enableAcceleratedNetworking` property that defaults to false in the apimodel and only enable it if that flag is set to true

Reply:
We could make it a property but I think it should be enabled by default because, AFAIU, accelerated networking as been made free to counter act the downsides of the Spectre/Meltdown patches on virtual networking.